examples/altair.html

```diff
@@ -56,7 +56,7 @@
 ).resolve_legend(
     color="independent",
     size="independent"
-))
+), target="altair")
```

Comment:
the `target=` is good, but now you can also remove the `output` attr in `<py-script output=...>`

Reply:
Oh good point thanks! I'll remove it in the next push

Reply:
👍